.codecov.yml

```diff
@@ -570,14 +574,14 @@ coverage:
         target: 75
         flags:
         - teamcity
-      Teleport:
-        target: 75
-        flags:
-        - teleport
       Tekton:
         target: 75
         flags:
         - tekton
+      Teleport:
+        target: 75
+        flags:
+        - teleport
```

Comment:
Could you undo all teleport-related changes? The best would be to make them in a separate PR, merge that to master and then update this PR.